services/github/github-common-release.js

```diff
@@ -68,8 +69,39 @@ function getLatestRelease({ releases, sort, includePrereleases }) {
 const queryParamSchema = Joi.object({
   include_prereleases: Joi.equal(''),
   sort: Joi.string().valid('date', 'semver').default('date'),
+  filter: Joi.string(),
 }).required()
 
+const filterDocs = `<div>
+  <p>
+  The <code>filter</code> param can be used to apply a filter to the
+  project's tag or release names before selecting the latest from the list.
+  Two constructs are available: <code>*</code> is a wildcard matching zero
+  or more characters, and if the pattern starts with a <code>!</code>,
+  the whole pattern is negated.
+  </p>
+</div>`
+
+function applyFilter({ releases, filter, displayName }) {
+  if (!filter) {
+    return releases
+  }
+  if (displayName === 'tag') {
```

Comment:
Here we apply the filter to either the release names or tag names depending on which is going to get displayed on the badge